### ISSUE TITLE
Remove xontrib mention

### ISSUE THREAD

**Issue by anki-code** [anki-code/xontrib-hist-format#2]
@daniel-shimon wrote:

> This is really cool! It seems that I can't get output without the links to the xontrib

**Comment by anki-code** [comment]
If you want to delete the mention you can do it manually. 

I'm not recommend to do it because: 

1. You found out about this extension when you saw the mention. [Don't be evil](https://en.wikipedia.org/wiki/Don%27t_be_evil) by removing it :)

2. The mission of this xontrib is to make commands repeatable, copy-pastable and save time for preparing - it's a good motivation to make people know about this xontrib.

3. You can suggest the better place for this mention.

4. I will remove the mention when this repo will have 10 stars. You can make step to this by clicking star ;)

**Comment by daniel-shimon** [comment]
Ok now without the link it looks nice, and the mention is good to have :)

**Comment by rpdelaney** [comment]
> You found out about this extension when you saw the mention. Don't be evil by removing it :)

I did not learn about it in this way. I learned about it by searching for xontribs on pypi. It would be nice if this could be disabled with a configuration env var. Removing it is not evil. Word of mouth recommendations are a better way to win adoption, and I'm less likely to recommend a xontrib if it has features I find annoying and can't disable.
